op-service/eth/id.go

```diff
@@ -85,6 +85,10 @@ func (id L1BlockRef) ParentID() BlockID {
 	}
 }
 
+// BlockRef is a Block Ref indepdendent of L1 or L2
+// Because L1BlockRefs are strict subsets of L2BlockRefs, BlockRef is a direct alias of L1BlockRef
```

Comment:
I would suggest describing `BlockRef` as the "the common properties of L1 and L2 blocks", to avoid any ambiguity where we might expect it to "work" as an L2 block in all cases.

Do you happen to not require the extra fields from `L2BlockRef` in op-supervisor, even when dealing with L2 blocks?

We could also consider embedding `BlockRef` into `L2BlockRef`, which could reduce duplication across a lot of the methods. But, this may result in some yak shaving (we would need to update hundreds of cases where we construct a literal `L2BlockRef`).